installer/cmd/render.go

```diff
@@ -30,6 +31,14 @@ var renderCmd = &cobra.Command{
 		for _, item := range yaml {
 			fmt.Println(item)
 		}
+		mixinImporter := importer.NewMixinImporter("https://github.com/gitpod-io/observability", "")
+		mixinImporter.ImportPrometheusRules()
+
+		yamlImporter := importer.NewYAMLImporter("https://github.com/ArthurSens/observability", "manifests/production/meta/kubescape", false)
+		yamlImporter.Import()
+
+		kustomizeImporter := importer.NewKustomizeImporter("https://github.com/kubernetes-sigs/kustomize", "examples/helloWorld")
+		kustomizeImporter.Import()
```

Comment:
if the imports are just examples and we don't actually want to import the stuff - would it be better to uncomment them?

Reply:
Do you mean comment? Yep that can be done 🙂

Reply:
yes - comment :)

Reply:
done!